spec/folio_sync/archives_space_to_folio/marc_downloader_spec.rb

```diff
@@ -70,7 +70,7 @@
       config = { aspace_to_folio: { marc_download_base_directory: '/base/dir' } }
       allow(Rails.configuration).to receive(:folio_sync).and_return(config)
```

Comment:
```suggestion
      config = { aspace_to_folio: { marc_download_base_directory: '/base/dir' } }
      allow(Rails.configuration).to receive(:folio_sync).and_return(config)
```
I think that with the latest version of this test, you can delete these two lines.